### PR TITLE
feat: add new grafana tiup pipeline

### PR DESCRIPTION
**Pull request by lijie0123**
# Why
- old pipeline embed logic, which is different with that in operator, we need to unify them
- pannel now managed within `monitoring` repo

# How
- use script in `pingcap/monitoring` repo
- when the script not exist in repo, we just fallback to old pipeline , and then cherry-pick build script to that branch. 
- without this fallback, a lot of pipeline would fail immediately, when most branch has been migrated, we can remove fallback

### CODE REVIEW

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy [hunk 1]
```diff
@@ -0,0 +1,109 @@
+def fallback=false
```
Comment (wuhuizuo) [inline jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy:1]: Why need this var to control the flow?

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            }
+            steps{
+                checkout([$class: 'GitSCM',
+                            branches: [[name: "${params.RELEASE_BRANCH}"]],
+                            extensions: [[$class: 'LocalBranch']],
+                            userRemoteConfigs: [[credentialsId: 'github-sre-bot-ssh', url: 'git@github.com:pingcap/monitoring.git']]]
+                )
+                script{
+                try{
+                    sh "bash scripts/prepare_dashboards.sh"
```
Comment (wuhuizuo) [inline jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy:40]: when it will failed?

Please add more description in pull request.

**Review by wuhuizuo — APPROVED** [review]

**Review by wuhuizuo — COMMENTED** [review]
please format it and add more descriptions and comments.

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                    options { retry(3) }
+                    environment { 
+                        OS="$OS"
+                        ARCH="$ARCH"
+                    }
+                    steps{
+                        container("tiup"){
+                            dir("$OS-$ARCH"){
+                            sh "bash ../scripts/build_tiup_grafana.sh"
+                            sh 'tiup mirror publish grafana ${VERSION} grafana.tar.gz "bin/grafana-server" --arch $ARCH --os $OS --desc="Grafana is the open source analytics & monitoring solution for every database" '
```
[the remark this answers is not in the document]
Comment (purelind) [reply jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy:100]: In this case, `tiup list` on darwin arm64 shows grafana 7.5.11, but the actual version is 7.5.10

[the remark this answers is not in the document]
Comment (lijie0123) [reply jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy:100]: let me refract this, action the version is tidb-version

File: jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy (hunk 1 above)
Comment (lijie0123) [reply jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update.groovy:1]: we have not cherry-pick script pr to all the branch, so we need this to fallback to old pipeline

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**